addons/account/models/ir_module.py

```diff
@@ -87,3 +87,10 @@ def _register_hook(self):
                 self.env.company,
             )
             del self.env.registry._auto_install_template
+
+    def module_uninstall(self):
+        unlinked_templates = [code for template in self.filtered('account_templates').mapped('account_templates') for code in template]
```

Comment:
Completely unnecessary, just makes the line longer
```suggestion
        unlinked_templates = [code for template in self.mapped('account_templates') for code in template]
```

Reply:
Done